docs/examples/frameworks/pytorch/pytorch-lightning.ipynb

```diff
@@ -325,8 +307,11 @@
     }
    ],
    "source": [
+    "# even if previous Trainer finihsed his work it still keeps the GPU booked, hack to free it\n",
```

Comment:
Just a nitpick, should this really be called a `hack`? Maybe
```suggestion
    "# Even if previous Trainer finished his work it still keeps the GPU booked, force it to release the device.\n",
```

+ there is a typo in `finished`. Same below.

Reply:
I don't think that changing the env variables set by lighting under the hood can be called anything else than a "hack".

Reply:
Fixed